### PR TITLE
Iterate tweet pages for database provisioning

### PR DESCRIPTION
**Pull request by Irio**
Fixes issue #1.

### CODE REVIEW

**Comment by jtemporal** [comment]
I believe Travis is failing because you are trying to clone Rosei with `ssh` try switching it over to `http` or `https` to see what happens.

**Comment by Irio** [comment]
@jtemporal Nope. Did not get it fixed it.

**Comment by Irio** [comment]
@jtemporal done.

File: whistleblower/targets/twitter.py [hunk 1]
```diff
@@ -80,23 +80,38 @@ def provision_database(self):
...
-        return self.api.GetUserTimeline(screen_name=self.PROFILE)
+        Posts already in the Twitter timeline. Paginates the results in lists
+        of 20 posts (given also to a Twitter API limitation).
+        """
+        max_id = None
+        while True:
+            posts = self.api.GetUserTimeline(screen_name=self.PROFILE,
+                                             max_id=max_id)
+            if max_id != None:
+                posts = posts[1:]
```
Comment (anaschwendler) [inline whistleblower/targets/twitter.py]: Twitter API only allows reading 20 statuses per time. And when you're about to read the next 20 statuses, you get the `id` of the last status on the previous list and the next 19 statuses. So you need to discard the first one when reading a new list.

**Comment by anaschwendler** [comment]
LGTM ;)

**Pull request merged**